src/index.html

```diff
@@ -116,73 +115,71 @@
     <textarea id="editor"
         placeholder="Ctrl+Enter to save.&#10;Type / to search.&#10;Drag & drop files to attach.&#10;Start links with + to save local copies."></textarea>
     <div id="submitContainer"><button id="submitButton">Submit</button></div>
-    <div id="notes">{{NOTES}}</div>
+    <div id="notes"></div>
 
     <script>
         const editor = document.getElementById('editor');
         const notesDiv = document.getElementById('notes');
         const submitButton = document.getElementById('submitButton');
         let searchTimeout = null;
-        let originalNotes = notesDiv.innerHTML;
 
-        // Check for search parameter on page load
-        window.addEventListener('load', () => {
-            const params = new URLSearchParams(window.location.search);
-            const searchQuery = params.get('q');
-            if (searchQuery) {
-                editor.value = '/' + decodeURIComponent(searchQuery);
-                performSearch(searchQuery);
-            }
+        window.addEventListener('load', async () => {
+            displayNotes();
         });
 
-        async function performSearch(query) {
-            const response = await fetch(`/search/${encodeURIComponent(query)}`);
+        // fetches and displays all notes, optionally filtering them based on the query parameter `q`
+        async function displayNotes() {
+            const params = new URLSearchParams(window.location.search);
+            const searchQuery = params.get('q');
+            let response = await fetch('/notes');
             if (response.ok) {
                 const notes = await response.json();
-                displayNotes(notes);
+                notesDiv.innerHTML = notes
+                    .filter(note => !searchQuery || note.content.includes(searchQuery))
```

Comment:
```suggestion
                    .filter(note => !searchQuery || note.content.toLowerCase().includes(searchQuery.toLowerCase()))
```

To replicate the previous behavior.